lib/App/Netdisco/Transport/SSH.pm

```diff
@@ -52,6 +52,11 @@ Returns C<undef> if the connection fails.
     my $self = shift;
     $self->platform->arpnip(@_, $self->host, $self->ssh, $self->auth);
   }
+
+  sub macsuck {
+    my $self = shift;
+    $self->plateform->macsuck(@_, $self->host, $self->ssh, $self->auth);
```

Comment:
has this code been tested? there's a typo here

Reply:
Currently, I use it for 2 devices on my test platform...
Which error message did you have ?
```
$ netdisco-do -D macsuck -d 10.12.32.61
[284503] 2024-04-15 19:56:53  info App::Netdisco version 2.072003 loaded.
[284503] 2024-04-15 19:56:53  info macsuck: [10.12.32.61] started at Mon Apr 15 21:56:53 2024
[284503] 2024-04-15 19:56:53 debug macsuck: running with timeout 600s
[284503] 2024-04-15 19:56:53 debug => running workers for phase: check
[284503] 2024-04-15 19:56:53 debug -> run worker check/1000000 "internal::backendfqdn"
[284503] 2024-04-15 19:56:53 debug -> run worker check/1000000 "internal::snmpfastdiscover"
[284503] 2024-04-15 19:56:53 debug running with configured SNMP timeouts
[284503] 2024-04-15 19:56:53 debug -> run worker check/0 "macsuck"
[284503] 2024-04-15 19:56:53 debug Macsuck is able to run.
[284503] 2024-04-15 19:56:53 debug => running workers for phase: early
[284503] 2024-04-15 19:56:53 debug -> run worker early/0 "prepare common data"
[284503] 2024-04-15 19:56:53 debug => running workers for phase: main
[284503] 2024-04-15 19:56:53 debug -> run worker main/1000000 "gather macs from file and set interfaces"
[284503] 2024-04-15 19:56:53 debug skip: fwtable data supplied by other source
[284503] 2024-04-15 19:56:53 debug -> run worker main/200 "gather macs from CLI and set interfaces"
[284503] 2024-04-15 19:56:53 debug cli session cache warm: [10.12.32.61]
[284503] 2024-04-15 19:56:54 debug 10.12.32.61 284503 macsuck()
[284503] 2024-04-15 19:56:56 debug -> run worker main/100 "gather macs from snmp and set interfaces"
[284503] 2024-04-15 19:56:56 debug skip: namespace passed at higher priority
[284503] 2024-04-15 19:56:56 debug -> run worker main/100 "macsuck::nodes::portaccessentity"
[284503] 2024-04-15 19:56:56 debug skip: namespace passed at higher priority
[284503] 2024-04-15 19:56:56 debug -> run worker main/100 "macsuck::wirelessnodes"
[284503] 2024-04-15 19:56:56 debug snmp reader cache warm: [10.12.32.61]
[284503] 2024-04-15 19:56:56 debug [10.12.32.61:161] try_connect with v: 3, t: 0.2, r: 0, class: SNMP::Info::Layer3::CiscoSwitch, comm: <hidden>
[284503] 2024-04-15 19:56:56 debug => running workers for phase: store
[284503] 2024-04-15 19:56:56 debug -> run worker store/0 "save macs to database"
[284503] 2024-04-15 19:56:56 debug  [10.12.32.61] macsuck 24:f2:7f:c7:5b:be - port GigabitEthernet0/1/0 has undiscovered neighbor 10.17.193.16
[284503] 2024-04-15 19:56:56 debug  [10.12.32.61] macsuck - port GigabitEthernet0/1/0 vlan 10 : 1 nodes
[284503] 2024-04-15 19:56:56 debug  [10.12.32.61] macsuck - stored 1 forwarding table entries
[284503] 2024-04-15 19:56:56 debug  [10.12.32.61] macsuck - removed 0 fwd table entries to archive
[284503] 2024-04-15 19:56:56 debug => running workers for phase: late
[284503] 2024-04-15 19:56:56 debug -> run worker late/0 "macsuck::hooks"
[284503] 2024-04-15 19:56:56 debug  [10.12.32.61] hooks - 0 queued
[284503] 2024-04-15 19:56:56  info macsuck: finished at Mon Apr 15 21:56:56 2024
[284503] 2024-04-15 19:56:56  info macsuck: status done: Ended macsuck for 10.12.32.61
```